src/test.hcl

```diff
@@ -1,5 +1,5 @@
 test "identifier store name conflict" {
-  err_contains = "Duplicate name"
+  err_contains = "duplicate name"
```

Comment:
think we'll want a `should_err = true/false` attribute too? so we can test valid lakefiles?

Reply:
if err_contains is "" or omitted it expects no errors

Reply:
which was actually not correctly implemented, just fixed that